app/modules/elasticsearch/tasks.py

```diff
@@ -95,3 +94,41 @@ def load_codex_indexes():
   left join "TAXONOMY" as tax on (mi."TAXONOMY_ID_OID" = tax."ID")
 ;
 """
+
+
+def load_encounters_index():
+    wb_engine = create_wildbook_engine()
+    with wb_engine.connect() as wb_conn:
+        results = wb_conn.execute(text(ENCOUNTERS_INDEX_SQL))
+        for result in results:
+            # Create the document object
+            encounter = Encounter(**result)
+            # Assign the elasticsearch document identify
+            encounter.meta.id = f'encounter_{encounter.id}'
+            # Save document to elasticsearch
+            encounter.save(using=current_app.elasticsearch)
+
+
+ENCOUNTERS_INDEX_SQL = """\
+SELECT
+  en."ID" AS id,
+  NULLIF((en."DECIMALLATITUDE"::float || ',' || en."DECIMALLONGITUDE")::text, ',') AS point,
```

Comment:
Think I'd favor something like 'gps' instead of 'point', but its not a big deal.

Reply:
The individuals search used "point" so I'm using it for the encounters and sightings search too.  It's definitely ok to change later though